# Translation of Odoo Server.
# This file contains the translation of the following modules:
# 	* account_yodlee
# 
# Translators:
# Martin Trigaux, 2019
# Chris Egal <sodaswed@web.de>, 2019
# 
msgid ""
msgstr ""
"Project-Id-Version: Odoo Server saas~12.5+e\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2019-08-26 08:04+0000\n"
"PO-Revision-Date: 2019-08-26 09:34+0000\n"
"Last-Translator: Chris Egal <sodaswed@web.de>, 2019\n"
"Language-Team: German (https://www.transifex.com/odoo/teams/41243/de/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: \n"
"Language: de\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid "(Diagnostic: %r for URL %s)"
msgstr "(Diagnostik: %r für URL %s)"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid "Account"
msgstr "Konto"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Account Already Registered (436):Your Auto-Registration could not be "
"completed because the site reports that your account is already registered."
"  Please log in to the site to confirm and then complete the site addition "
"process with the correct login information."
msgstr ""
"Konto bereits registriert (436): Ihre Auto-Registrierung wurde nicht "
"abgeschlossen, da Ihr Konto von der Seite als bereits registriert berichtet "
"wird. Bitte melden Sie sich zum Bestätigen bei der Seite an und schließen "
"Sie dann den das Hinzufügen mit den korrekten Anmeldeinformationen ab."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Account Locked (407): We could not update your account because it appears "
"your account has been locked. This usually results from too many "
"unsuccessful login attempts in a short period of time. Please visit the site"
" or contact its customer support to resolve this issue.  Once done, please "
"update your account credentials in case they are changed."
msgstr ""
"Konto gesperrt (407): Wir konnten Ihr Konto nicht aktualisieren, weil es "
"gesperrt ist. Meist geschieht dies durch zu häufige Anmeldeversuche in "
"kurzer Zeit. Bitte besuchen Sie die Seite oder kontaktieren Sie zur "
"Problembehebung den Kundensupport. Anschließend können Sie Ihre "
"Konteninformationen ändern, falls sich diese geändert haben."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Account Not Found(408): We're sorry, we couldn't find any accounts for you "
"at the site. Please log in at the site and confirm that your account is set "
"up, then try again."
msgstr ""
"Konto nicht gefunden (408): Es tut uns leid, wir haben von Ihnen kein Konto "
"auf der Seite gefunden. Bitte melden Sie sich bei der Seite an und "
"bestätigen Sie, dass Ihr Konto eingerichtet ist. Versuchen Sie es "
"anschließend erneut."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Account Reported Closed (422): We were unable to update your account "
"information because it appears one or more of your related accounts have "
"been closed.  Please deactivate or delete the relevant account and try "
"again."
msgstr ""
"Konto geschlossen (422): Wir konnten Ihre Konteninformationen nicht "
"aktualisieren, weil eines oder mehrere Ihrer zugehörigen Konten geschlossen "
"wurden.  Bitte deaktivieren oder löschen Sie das relevante Konto und "
"versuchen Sie es erneut."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Account Type Not Supported(417):The type of account we found is not "
"currently supported.  Please remove this site and add as a  manual account."
msgstr ""
"Kontotyp nicht unterstützt(417): Der Typ des gefundenen Kontos ist zurzeit "
"nicht unterstützt. Bitte löschen Sie diese Seite und fügen es als manuelles "
"Konto hinzu."

#. module: account_yodlee
#: model:ir.model.fields,help:account_yodlee.field_account_online_journal__yodlee_account_status
msgid "Active/Inactive on Yodlee system"
msgstr "Aktiv/Inaktiv auf Yodlee System"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid "An Error has occurred (code %s)"
msgstr "Ein Fehler ist aufgetreten (code %s)"

#. module: account_yodlee
#: model:ir.model,name:account_yodlee.model_res_company
msgid "Companies"
msgstr "Unternehmen"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid "Could not make sense of parameters: %s"
msgstr "Es konnte den  Parameter nicht verstehen: %s"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Credential Re-Verification Required (402): We could not update your account "
"because your username and/or password were reported to be incorrect.  Please"
" re-verify your username and password."
msgstr ""
"Erneutes Verifizieren von Angaben erforderlich (402): Wir konnten Ihr Konto "
"nicht aktualisieren, weil Ihr Benutzername und/oder Passwort als inkorrekt "
"berichtet wurden. Bitte verifizieren Sie Benutzername und Passwort erneut."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Credential Re-Verification Required (420):The site has merged with another. "
"Please re-verify your credentials at the site and update the same."
msgstr ""
"Erneute Verifizierung von Angaben erforderlich (420): Die Seite wurde mit "
"einer anderen zusammengelegt. Bitte verifizieren Sie Ihre Angaben auf der "
"Seite erneut aktualisieren Sie diese."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid "Error %s, message: %s"
msgstr "Fehler %s, Nachricht: %s"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid "Error %s, message: %s, reference code: %s"
msgstr "Fehler %s, Nachricht: %s, Referenzcode: %s"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Home Value Not Found (511): The site is unable to provide home value for "
"your property. We suggest you to delete this site."
msgstr ""
"Heimwert nicht gefunden (511): Die Seite kann für Ihr Eigentum keinen "
"Heimwert angeben. Wir empfehlen, diese Seite zu löschen."

#. module: account_yodlee
#: model:ir.model,name:account_yodlee.model_account_online_journal
msgid "Interface for Online Account Journal"
msgstr "Schnittstelle für Online Account Journal"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Invalid Language Setting (421): The language setting for your site account "
"is not English. Please visit the site and change the language setting to "
"English."
msgstr ""
"Ungültige Spracheinstellung (421): Die Spracheinstellung für Ihr Seitenkonto"
" ist nicht Englisch. Bitte besuchen Sie die Seite und ändern Sie die "
"Einstellung auf Englisch."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"MFA error: Additional Authentication Enrollment Required (521) : Please "
"enroll in the new security authentication system, <Account Name> has "
"introduced. Ensure your account settings in <Cobrand> are updated with this "
"information."
msgstr ""
"MFA-Fehler: Zusätzliche Authentifizierungsregistrierung erforderlich (521): "
"Bitte registrieren Sie sich im neuen Sicherheitsauthentifizierungssystem, "
"das <Account Name> eingeführt hat. Stellen Sie sicher, dass Ihre "
"Kontoeinstellungen bei <Cobrand> mit diesen Informationen aktualisiert "
"wurden."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"MFA error: Authentication Information Expired (524):We're sorry, the "
"authentication information you provided has expired. Please try again."
msgstr ""
"MFA-Fehler: Authentifizierungsinformationen abgelaufen (524): Leider sind "
"die von Ihnen angegebenen Authentifizierungsinformationen abgelaufen. Bitte "
"versuchen Sie es erneut."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"MFA error: Authentication Information Incorrect (520):We're sorry, the site "
"indicates that the additional authentication information you provided is "
"incorrect. Please try updating your account again."
msgstr ""
"MFA-Fehler: Authentifizierungsinformationen ungültig (520): Leider zeigt die"
" Seite an, dass die von Ihnen angegebenen zusätzlichen "
"Authentifizierungsinformationen ungültig sind. Bitte versuchen Sie, Ihr "
"Konto erneut zu aktualisieren."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"MFA error: Authentication Information Incorrect (523):We're sorry, the "
"authentication information you  provided is incorrect. Please try again."
msgstr ""
"MFA-Fehler: Authentifizierungsinformationen ungültig (523):We're sorry, the "
"authentication information you  provided is incorrect. Please try again."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"MFA error: Authentication Information Required (519): Your account was not "
"updated as your authentication information like security question and answer"
" was unavailable or incomplete. Please update your account settings."
msgstr ""
"MFA-Fehler: Authentifizierungsinformationen erforderlich (519): Ihr Konto "
"wurde nicht aktualisiert, da Authentifizierungsinformationen wie "
"Sicherheitsfrage und -antwort nicht vorhanden oder unvollständig sind. Bitte"
" aktualisieren Sie Ihre Kontoeinstellungen."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"MFA error: Authentication Information Unavailable (518):Your account was not"
" updated as the required additional authentication information was "
"unavailable. Please try now."
msgstr ""
"MFA-Fehler: Authentifizierungsinformationen nicht verfügbar (518): Ihr Konto"
" wurde nicht aktualisiert, da die erforderlichen zusätzlichen "
"Authentifizierungsinformationen nicht verfügbar sind. Bitte versuchen Sie es"
" jetzt erneut."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"MFA error: Credential Re-Verification Required (526): We could not update "
"your account because your username/password or additional security "
"credentials are incorrect. Please try again."
msgstr ""
"MFA-Fehler: Erneutes Verifizieren von Angaben erforderlich (526): Ihr Konto "
"wurde aktualisiert, weil Benutzername und/oder Passwort ungültig sind. Bitte"
" versuchen Sie es erneut."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"MFA error: Problem Updating Account (525): We could not update your account "
"for technical reasons. This type of error is usually resolved in a few days."
" We apologize for the inconvenience. Please try again later."
msgstr ""
"MFA-Fehler: Problem Updating Account (525): We could not update your account"
" for technical reasons. This type of error is usually resolved in a few "
"days. We apologize for the inconvenience. Please try again later."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"MFA error: Request Timed Out (522) :Your request has timed out as the "
"required security information was unavailable or was not provided within the"
" expected time. Please try again."
msgstr ""
"MFA-Fehler: Anfrage abgelaufen (522): Ihre Anfrage ist abgelaufen, da die "
"erforderlichen Sicherheitsinformationen nicht im dafür angelegten Zeitraum "
"abgerufen werden konnten. Bitte versuchen Sie es erneut."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"MFA error: Site Device Information Expired(509): We're sorry, we can't "
"update your account because your token is no longer valid at the site. "
"Please update your information and try again, or contact customer support."
msgstr ""
"MFA-Fehler: Geräteinformationen der Seite abgelaufen (509): We're sorry, we "
"can't update your account because your token is no longer valid at the site."
" Please update your information and try again, or contact customer support."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Multiple User Logins(416): We attempted to update your account, but another "
"session was already established at the same time.  If you are currently "
"logged on to this account directly, please log off and try after some time"
msgstr ""
"Multiple Benutzer-Logins (416): Beim Aktualisieren Ihres Kontos war eine "
"andere Sitzung aktiv. Wenn Sie momentan direkt bei diesem Konto angemeldet "
"sind, melden Sie sich ab und versuchen Sie es nach einer Weile erneut."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"New Login Information Required(506):We're sorry, to log in to this site, you"
" need to provide additional information. Please update your account and try "
"again."
msgstr ""
"Neue Anmeldeinformationen erforderlich (506): Leider sind für das Anmelden "
"bei dieser Seite zusätzliche Informationen erforderlich. Please update your "
"account and try again."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"No Account Found (423): We were unable to detect an account. Please verify "
"that you account information is available at this time and If the problem "
"persists, please contact customer support at online@odoo.com for further "
"assistance."
msgstr ""
"Kein Konto gefunden (423): Es wurde kein Konto gefunden. Bitte verifizieren "
"Sie, dass Ihre Konteninformationen momentan verfügbar sind. Wenn das Problem"
" weiterhin besteht, kontaktieren Sie bitte für weitere Hilfe den "
"Kundensupport unter online@odoo.com."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"No Payees Found(512):Your request cannot be completed as no payees were "
"found in your account."
msgstr ""
"Keine Zahlungsempfänger gefunden (512): Ihre Anfrage kann nicht "
"abgeschlossen werden, da unter Ihrem Konto keine Zahlungsempfänger gefunden "
"wurden."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid "Online institution"
msgstr "Online-Institution"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid "Please configure an email in the company settings."
msgstr "Bitte hinterlegen Sie eine E-Mail in den Unternehmeseinstellungen"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account (406): We could not update your account because the"
" site requires you to perform some additional action. Please visit the site "
"or contact its customer support to resolve this issue. Once done, please "
"update your account credentials in case they are changed else try again."
msgstr ""
"Problem beim Aktualisieren des Kontos (406): Ihr Konto konnte nicht "
"aktualisiert werden, da die Seite eine zusätzliche Aktion erfordert. Bitte "
"besuchen Sie die Seite oder kontaktieren Sie den Kundensupport to resolve "
"this issue. Once done, please update your account credentials in case they "
"are changed else try again."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account (517): We are sorry, there was a technical problem "
"updating your account. Please try again."
msgstr ""
"Problem bei der Aktualisierung Ihres Kontos (517): Es tut uns leid, bei der "
"Aktualisierung Ihres Kontos ist ein technisches Problem aufgetreten. Bitte "
"versuche es erneut."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account(401):We're sorry, your request timed out. Please "
"try again."
msgstr ""
"Problem beim Aktualisieren des Kontos (401): Leider ist Ihre Anfrage "
"abgelaufen. Bitte versuchen Sie es erneut."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account(403):We're sorry, there was a technical problem "
"updating your account. This kind of error is usually resolved in a few days."
" Please try again later."
msgstr ""
"Problem beim Aktualisieren des Kontos (403): Leider ist beim Aktualisieren "
"Ihres Kontos ein technisches Problem aufgetreten. Dieser Fehlertyp ist "
"häufig innerhalb von wenigen Tagen aufgehoben. Bitte versuchen Sie es später"
" erneut."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account(404):We're sorry, there was a technical problem "
"updating your account. Please try again later."
msgstr ""
"Problem beim Aktualisieren des Kontos (404): Leider ist beim Aktualisieren "
"Ihres Kontos ein technisches Problem aufgetreten. Bitte versuchen Sie es "
"später erneut."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account(409): We could not update your account because the "
"end site is experiencing technical difficulties."
msgstr ""
"Problem beim Aktualisieren des Kontos (409): Ihr Konto konnte nicht "
"aktualisiert werden, weil "

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account(412): We could not update your account because the "
"site is experiencing technical difficulties."
msgstr ""
"Problem beim Aktualisieren des Kontos (412): We could not update your "
"account because the site is experiencing technical difficulties."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account(413):We're sorry, we couldn't update your account "
"at the site because of a technical issue. This type of problem is usually "
"resolved in a few days. Please try again later."
msgstr ""
"Problem beim Aktualisieren des Kontos (413):We're sorry, we couldn't update "
"your account at the site because of a technical issue. This type of problem "
"is usually resolved in a few days. Please try again later."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account(415): We could not update your account because the "
"site is experiencing technical difficulties."
msgstr ""
"Problem beim Aktualisieren des Kontos (415): Ihr Konto konnte aufgrund "
"technischer Schwierigkeiten der Seite nicht aktualisiert werden."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account(418): We could not update your account because the "
"site is experiencing technical difficulties. Please try later."
msgstr ""
"Problem beim Aktualisieren des Kontos (418): Aufgrund technischer "
"Schwierigkeiten der Seite konnte Ihr Konto nicht aktualisiert werden. Bitte "
"versuchen Sie es erneut."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account(419):We're sorry, we couldn't update your account "
"because of unexpected variations at the site. This kind of problem is "
"usually resolved in a few days. Please try again later."
msgstr ""
"Problem beim Aktualisieren des Kontos (419): Leider konnten wir Ihr Konto "
"aufgrund von unerwarteten Variationen auf der Seite nicht aktualisieren. "
"Dieser Problemtyp ist häufig innerhalb von wenigen Tagen aufgehoben. Bitte "
"versuchen Sie es später erneut."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account(425): We could not update your account because the "
"site is experiencing technical difficulties. Please try later."
msgstr ""
"Problem beim Aktualisieren des Kontos (425): Aufgrund technischer "
"Schwierigkeiten der Seite konnte Ihr Konto nicht aktualisiert werden. Bitte "
"versuchen Sie es später."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account(426): We could not update your account for "
"technical reasons. This type of error is usually resolved in a few days. We "
"apologize for the inconvenience."
msgstr ""
"Problem beim Aktualisieren des Kontos (426): Aufgrund technischer "
"Schwierigkeiten konnte Ihr Konto nicht aktualisiert werden. Dieser Fehlertyp"
" ist häufig innerhalb von wenigen Tagen aufgehoben. Wir entschuldigen uns "
"für etwaige Unannehmlichkeiten."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account(507):We're sorry, Yodlee has just started providing"
" data updates for this site, and it may take a few days to be successful as "
"we get started. Please try again later."
msgstr ""
"Problem beim Aktualisieren des Kontos (507):We're sorry, Yodlee has just "
"started providing data updates for this site, and it may take a few days to "
"be successful as we get started. Please try again later."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Property Record Not Found (510): The site is unable to find any property "
"information for your address. Please verify if the property address you have"
" provided is correct."
msgstr ""
"Adressangabe nicht gefunden (510): Die Seite kann Ihre Adresse nicht "
"verifizieren. Bitte bestätigen Sie die Korrektheit Ihrer Angaben.."

#. module: account_yodlee
#: model:ir.model.fields,field_description:account_yodlee.field_account_online_provider__provider_type
msgid "Provider Type"
msgstr "Anbietertyp"

#. module: account_yodlee
#: model:ir.model,name:account_yodlee.model_account_online_provider
msgid "Provider for online account synchronization"
msgstr "Anbieter für online Konto Synchronisation"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Re-Verification Required (429): We could not update your account due to the "
"site requiring you to verify your personal information. Please log in to the"
" site and update the fields required."
msgstr ""
"Erneutes Verifizieren von Angaben erforderlich (429): Wir konnten Ihr Konto "
"nicht aktualisieren, weil die Seite das Verifizieren Ihrer persönlichen "
"Informationen erfordert. Bitte melden Sie sich bei der Seite an und "
"aktualisieren Sie die erforderlichen Felder."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Re-verification Required (427): We could not update your account due to the "
"site requiring you to view a new promotion. Please log in to the site and "
"click through to your account overview page to update the account.  We "
"apologize for the inconvenience."
msgstr ""
"Erneutes Verifizieren von Angaben erforderlich (427): We could not update "
"your account due to the site requiring you to view a new promotion. Please "
"log in to the site and click through to your account overview page to update"
" the account.  We apologize for the inconvenience."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Re-verification Required (428): We could not update your account due to the "
"site requiring you to accept a new Terms & Conditions. Please log in to the "
"site and read and accept the T&C."
msgstr ""
"Erneutes Verifizieren von Angaben erforderlich (428): Ihr Konto konnte nicht"
" aktualisiert werden, da die Seite das Akzeptieren neuer Nutzungsbedingungen"
" erfordert. Please log in to the site and read and accept the T&C."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Registration Requires Attention (433): Auto registration is not complete. "
"Please complete your registration at the end site. Once completed, please "
"complete adding this account."
msgstr ""
"Registrierung erforderlich (433): Die Auto-Registrierung ist nicht "
"vollständig. Bitte vervollständigen Sie Ihre Registrierung bei der Endseite."
" Bitte schließen Sie nach dem Fertigstellen das Hinzufügen dieses Kontos ab."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Registration Requires Attention (434): Your Auto-Registration could not be "
"completed and requires further input from you.  Please re-verify your "
"registration information to complete the process."
msgstr ""
"Registrierung erforderlich (434): Ihre Auto-Registrierung konnte nicht "
"abgeschlossen werden und erfordert eine Eingabe von Ihnen. Bitte "
"verifizieren Sie Ihre Registrierungsinformationen erneut, um diesen Prozess "
"abzuschließen."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Registration Requires Attention (435): Your Auto-Registration could not be "
"completed and requires further input from you.  Please re-verify your "
"registration information to complete the process."
msgstr ""
"Registrierung erforderlich (435): Ihre Auto-Registrierung konnte nicht "
"abgeschlossen werden und erfordert eine Eingabe von Ihnen. Bitte "
"verifizieren Sie Ihre Registrierungsinformationen erneut, um diesen Prozess "
"abzuschließen."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Request Timed Out (508): We are sorry, your request timed out due to "
"technical reasons. Please try again."
msgstr ""
"Anfrage abgelaufen (508): Leider ist Ihre Anfrage aus technischen Gründen "
"abgelaufen. Bitte versuchen Sie es erneut."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Requested Account Type Not Found (414): We could not find your requested "
"account. You may have selected a similar site under a different category by "
"accident in which case you should select the correct site."
msgstr ""
"Angefragter Kontotyp nicht gefunden (414): Das angefragte Konto wurde nicht "
"gefunden. Möglicherweise haben Sie aus Versehen eine ähnliche Seite einer "
"anderen Kategorie gewählt. Bitte wählen Sie die korrekte Seite."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid "Server not reachable, please try again later"
msgstr "Server nicht erreichbar, bitte versuchen Sie es später erneut"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Site Down for Maintenance(424):We were unable to update your account as the "
"site is temporarily down for maintenance. We apologize for the "
"inconvenience.  This problem is typically resolved in a few hours. Please "
"try later."
msgstr ""
"Seite aufgrund von Wartung nicht verfügbar (424): Ihr Konto konnte nicht "
"aktualisiert werden, da die Seite aufgrund von Wartung vorübergehend nicht "
"verfügbar ist. Entschuldigen Sie bitte etwaige Unannehmlichkeiten. Dieses "
"Problem ist häufig innerhalb von wenigen Stunden aufgehoben. Bitte versuchen"
" Sie es später erneut."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Site No Longer Available (411):The site no longer provides online services "
"to its customers.  Please delete this account."
msgstr ""
"Seite nicht mehr verfügbar (411): Die Seite bietet ihren Kunden keine "
"Online-Dienste mehr an. Bitte löschen Sie dieses Konto."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Site No Longer Supported (430):This site is no longer supported for data "
"updates. Please deactivate or delete your account. We apologize for the "
"inconvenience."
msgstr ""
"Seite nicht mehr unterstützt (430): DIese Seite wird nicht mehr für "
"Datenaktualisierungen unterstützt. Bitte deaktivieren oder löschen Sie Ihr "
"Konto. Wir entschuldigen uns für etwaige Unannehmlichkeiten."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Site Not Supported (505): We currently does not support the security system "
"used by this site. We apologize for any inconvenience. Check back "
"periodically if this situation has changed."
msgstr ""
"Seite nicht unterstützt (505): Derzeit unterstützen wir das "
"Sicherheitssystem dieser Seite nicht. Entschuldigen Sie bitte etwaige "
"Unannehmlichkeiten. Prüfen Sie die Seite ab und zu auf eine Änderung dieser "
"Situation."

#. module: account_yodlee
#: model:ir.actions.client,name:account_yodlee.yodlee_widget
msgid "Synchronization process completed"
msgstr "Synchronisationsprozess abgeschlossen"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#: code:addons/account_yodlee/models/yodlee.py:0
#: code:addons/account_yodlee/models/yodlee.py:0
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid "Timeout: the server did not reply within 30s"
msgstr ""
"Zeitüberschreitung: Der Server hat innerhalb von 30s nicht geantwortet"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Update Request Canceled(405):Your account was not updated because you "
"canceled the request."
msgstr ""
"Aktualisierungsanfrage abgebrochen (405): Ihr Konto wurde nicht "
"aktualisiert, da Sie die Anfrage abgebrochen haben."

#. module: account_yodlee
#: model:ir.model.fields.selection,name:account_yodlee.selection__account_online_provider__provider_type__yodlee
msgid "Yodlee"
msgstr "Yodlee"

#. module: account_yodlee
#: model:ir.model.fields,field_description:account_yodlee.field_account_online_journal__yodlee_account_status
msgid "Yodlee Account Status"
msgstr "Yodlee Kontostatus"

#. module: account_yodlee
#: model:ir.model.fields,field_description:account_yodlee.field_account_online_journal__yodlee_status_code
msgid "Yodlee Status Code"
msgstr "Yodlee Statuscode"

#. module: account_yodlee
#: model:ir.model.fields,field_description:account_yodlee.field_res_company__yodlee_user_access_token
msgid "Yodlee access token"
msgstr "Yodlee Zugangstoken"

#. module: account_yodlee
#: model:ir.model.fields,field_description:account_yodlee.field_res_company__yodlee_user_login
msgid "Yodlee login"
msgstr "Yodlee Login"

#. module: account_yodlee
#: model:ir.model.fields,field_description:account_yodlee.field_res_company__yodlee_user_password
msgid "Yodlee password"
msgstr "Yodlee Passwort"

#. module: account_yodlee
#: model:ir.model.fields,field_description:account_yodlee.field_res_company__yodlee_access_token
msgid "access_token"
msgstr "Zugangstoken"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid "message: %s"
msgstr "Nachricht: %s"
